app/src/scenes/public-support-center/form.js

```diff
@@ -25,12 +25,13 @@ export default ({ setOpen, setSuccessMessage }) => {
           try {
             setLoading(true);
             const { message, subject, name, email, step1, step2 } = values;
-            const { ok, code, data } = await api.post("/support-center/public/ticket", {
+            const { ok, code } = await api.post("/support-center/public/ticket", {
               title: `${step1?.label} - ${step2?.label} - ${subject}`,
               subject,
               name,
               email,
               message,
+              // eslint-disable-next-line no-unsafe-optional-chaining
               tags: [...new Set([...tags, ...step1?.tags, ...step2?.tags])], // we use this dirty hack to remove duplicates
```

Comment:
Là c'est vraiment un bug potentiel, je conseillerait de s'assurrer qu'on a un tableau. ATtention, code non testé.

```suggestion
              tags: [...new Set([...tags, ...[step1?.tags || []], ...[step2?.tags || []]])], // we use this dirty hack to remove duplicates
```

Reply:
Ca fonctionne, je valide